tables/output_tables.Rmd

```diff
@@ -344,9 +355,80 @@ write.xlsx(list_s3_table,
 
 ```
 
-## Table S4: `Key Resources` table
-
+## Table S4: `Survival results` table
 ```{r table_s4}
+# Read in files
+cox_subtypes <- readRDS(cox_subtypes_file)
+lr_subtypes <- readRDS(lr_subtypes_file)
+cox_tp53 <- readRDS(cox_tp53_file)
+cox_dmg_tp53 <- readRDS(cox_dmg_tp53_file)
+cox_epn <- readRDS(cox_epn_tp53_file)
+cox_mb_quantiseq <- readRDS(cox_mb_quantiseq_file)
+
```

Comment:
We should use a function here since there's a lot of repeated code. Something like this:

```process_survival <- function(fit) {
broom::tidy(fit) %>%
 mutate(HR = exp(estimate),
         conf.low = exp(conf.low),
         conf.high = exp(conf.high)
  ) %>%
  select(-estimate)
}

# And use as:
cox_subtypes_df <- process_survival(cox_subtypes)
```